tests/features/event.feature

```diff
@@ -269,3 +269,61 @@ Feature: Event content type.
+      | Address      | country_code: HU - locality: Szeged - address_line1: Press contact 1 - postal_code: 6700 |
+      | Email        | press2@example.com                                                                       |
+      | Phone number | +32477777778                                                                             |
+    And the following General Contact entity:
+      | Name         | A general contact                                                                            |
+      | Address      | country_code: HU - locality: Budapest - address_line1: General contact 1 - postal_code: 1011 |
+      | Email        | general@example.com                                                                          |
+      | Phone number | +32477792933                                                                                 |
+    And the following Event Content entity:
+      | Title   | Event demo page                                              |
+      | Venue   | DIGIT                                                        |
+      | Contact | First press contact, Second press contact, A general contact |
+    When I am visiting the "Event demo page" content
+    Then I should not see the text "DIGIT"
+    And I should not see the text "First press contact"
+    And I should not see the text "Second press contact"
+    And I should not see the text "A general contact"
+    When anonymous users can see events
+    And I reload the page
+    Then I should see the text "DIGIT"
+    And I should see the text "First press contact"
+    And I should see the text "Second press contact"
+    And I should see the text "General contact"
+
+  @preserve_anonymous_permissions
+  Scenario: Unpublished Venues and Contacts are not visible for the visitors.
+    Given anonymous users can see events
+    And the following Default Venue entity:
+      | Name    | Unpublished venue                                                                          |
+      | Address | country_code: BE - locality: Brussels - address_line1: Rue Belliard 28 - postal_code: 1000 |
+      | status  | 0                                                                                          |
+    And the following Event Content entity:
+      | Title | Event demo page   |
+      | Venue | Unpublished venue |
+    When I am visiting the "Event demo page" content
+    Then I should not see the text "Unpublished venue"
+    When the following Default Venue entity:
+      | Name    | Published venue                                                                            |
+      | Address | country_code: BE - locality: Brussels - address_line1: Rue Belliard 28 - postal_code: 1000 |
+    And the Event Content "Event demo page" is updated as follows:
+      | Venue | Published venue |
+    And I am visiting the "Event demo page" content
+    Then I should see the text "Published venue"
```

Comment:
Missing the checks for the entitiy items.

Reply:
For the contact items you mean :)